system/core/pages.php

```diff
@@ -534,3 +567,60 @@ function image_resize($src, $dst, $width, $height) {
 		}
 		return true;
 	}
+
+	function getButtonInfo($_args) {
+		/* 0 = buttonName
+		*  1 = buttonName
+		*  2 = language (or null)
+		*/
+		Global $O_O;
+		$_button = "";
+		$_language = "";
+
+		if (sizeof($_args) > 1) {$_button = $_args[1];} else {$_button = "undifined";}
+		$a = array("%20","%22");
+		$b = array(" "," ");
+	  $_button = str_replace($a,$b,$_button);
+		if (sizeof($_args) > 2) {$_language = $_args[2];} else {$_language=$O_O->getInterfaceLanguage();} //$_language=$O_O->getContentLanguage(); //DefaultLanguage();
+		$info = hypha_searchhelp($_button,$_language);
+		return  $info ; // "AJAX - Help voor <br>" . $_button . "<br> in de taal: ". $_language;
+	}
+
+	function hypha_isLanguage($id) {
+		$pageLangList = array("nl","en","de"); // replace by hypha language list
```

Comment:
This code is unfinished. IIRC HyphaRequest already has a similar check, which could be reused (or moved here, and reused by HyphaRequest). In any case, the addition of this function would be better placed in a separate commit, since it is not tied to the tooltip stuff (this function could very well be used by other code too).